pyproject.toml

```diff
@@ -7,20 +7,20 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "3.8.12"
-Django = "*"
-python-dotenv = "*"
-django-heroku = "*"
-google-api-python-client = "*"
-social-auth-app-django = "*"
-django-taggit = "*"
-url-normalize = "*"
-gunicorn = "*"
-djangorestframework = "*"
+Django = "^4.0.1"
+python-dotenv = "^0.19.2"
+django-heroku = "^0.3.1"
+google-api-python-client = "^2.33.0"
+social-auth-app-django = "^5.0.0"
+django-taggit = "^2.0.0"
+url-normalize = "^1.4.3"
+gunicorn = "^20.1.0"
+djangorestframework = "^3.13.1"
 disco-py = ">=0.0.14rc1"
-celery = "*"
-django-celery-results = "*"
-redis = "*"
-"discord.py" = "*"
+celery = "^5.2.3"
+django-celery-results = "^2.2.0"
+redis = "^4.1.0"
+"discord.py" = "^1.7.3"
```

Comment:
Can you use an asterisk for part of the version, e.g.: `^4.*`? Might be nice to automatically pick up minor and patch version bumps. (I think this would also reduce the number of Dependabot security version bump PRs we get.) Though using floating minor and patch versions might also cause the poetry.lock file to change frequently and cause unnecessary noise in PRs. What do you guys think?

Reply:
That's what the ^ does: https://python-poetry.org/docs/dependency-specification/ So eg `^4.0.1` means any 4.x.y version that's  >=4.0.1. The only difference is that it excludes `4.0.0` specifically. Both probably work for us I think.

Reply:
oooh got it now
Thanks for explaining.